suites/reef/cephfs/tier-3_cephfs_bugs.yaml

```diff
@@ -252,3 +252,10 @@ tests:
       polarion-id: CEPH-83591136
       desc: Verify Ceph Health Warning for Slow MDS Requests After Cluster Installation
       abort-on-fail: false
+  - test:
```

Comment:
nit: ideally, I would expect this to be a separate commit. This helps in identifying changes.

Reply:
done, removed suite files, will include it another PR